lib/ember_cli/runner.rb

```diff
@@ -32,9 +35,11 @@ def run!(command)
 
     private
 
-    def write(output, streams:)
-      streams.each do |stream|
-        stream.write(output)
+    def redirect(stream, pipes:)
+      Tee.open(*pipes, stdout: nil) do |pipe|
+        while line = stream.gets
```

Comment:
@bernerdschaefer this PR is a follow up to https://github.com/thoughtbot/ember-cli-rails/pull/420.

The idea here is that streaming up-to-date information is better than waiting until the subprocess completes, then dumping all output.

Do you have suggestions for a better approach [that also handles `STDOUT` / `STDERR` redirection from Ruby-land](https://github.com/thoughtbot/ember-cli-rails/issues/417#issuecomment-197991094)?